src/components/sync-connected-sites.tsx

```diff
@@ -71,6 +72,15 @@ export function SyncConnectedSites( {
 		return siteSections;
 	}, [ connectedSites ] );
 
+	const showPullConfirmation = useConfirmationDialog( {
+		localStorageKey: 'dontShowPullConfirmation',
+		message: __( 'Overwrite Studio site' ),
+		detail: __(
+			"Pulling will replace your Studio site's files and database with a copy from your staging site."
```

Comment:
It seems that at the moment, I am getting the text "from your staging site" for both staging and production sites. Should we change it to "from your production site" for production sites?

Reply:
@katinthehatsite, I added both description on https://github.com/Automattic/studio/pull/663/commits/cbf5e15e4571c4fffed7385764fbe0cf6b0cc956. Thanks for pointing this out.